### DISCUSSION TITLE
Button for equipping changed items from mod

### DISCUSSION THREAD

**Discussion by Hapseleg** [xivdev/Penumbra#338] [Ideas]
A simple button that unequip all your current items and instead equips the changed items in the mod. 
A small problem is that some mods have multiple items in the same item slot.. To keep it simple I'd say just use the first one of each type and ignore the rest.

![image](https://github.com/xivdev/Penumbra/assets/12915302/581debed-06e2-4ded-aa83-aff0df6ed636)

**Comment by Ottermandias** [comment]
That's not even a Penumbra feature at all, so no.
